src/subcommand/wallet/send.rs

```diff
@@ -0,0 +1,47 @@
+use {
+  super::*, bitcoin::util::amount::Amount, bitcoincore_rpc::json::CreateRawTransactionInput,
+  std::collections::HashMap,
+};
+
+#[derive(Debug, Parser)]
+pub(crate) struct Send {
+  ordinal: Ordinal,
+  address: String,
+}
+
+impl Send {
+  pub(crate) fn run(self, options: Options) -> Result {
+    if options.chain.network() == Network::Bitcoin {
+      bail!("Send command is not allowed on mainnet yet. Try on regtest/signet/testnet.")
```

Comment:
Also check that the bitcoin core node that we're connected to is on the right network using `getblockchaininfo` and refuse to work if it's `mainnet`.